migrations/20230310-create-init-schema.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE users (
+	id serial primary key,
+	name varchar(150) NOT NULL,
+	email varchar(120) NOT NULL,
+	phone_number varchar(18),
+	password_hash varchar(72) NOT NULL
+);
+
+CREATE TABLE resources (
+	id serial primary key,
+	location geometry NOT NULL,
+	type varchar(60) NOT NULL,
+	name varchar(120),
+	quantity int NOT NULL CHECK (quantity > 0),
+	owned_by int,
+	reserved_by int,
+	foreign key (owned_by) references users(id),
+	foreign key (reserved_by) references users(id)
+);
+
+CREATE TABLE images (
+	id serial primary key,
+	content varchar(300) NOT NULL,
```

Comment:
Did we decide how to store images in the db?

Reply:
I think we decided on URLs https://github.com/LumosCapstone/backend/pull/9#issuecomment-1460679172